maqs-selenium/src/test/java/com/cognizantsoftvision/maqs/selenium/pageModel/HeaderPageModel.java

```diff
@@ -4,10 +4,7 @@
 
 package com.cognizantsoftvision.maqs.selenium.pageModel;
 
-import com.cognizantsoftvision.maqs.selenium.BaseSeleniumPageModel;
-import com.cognizantsoftvision.maqs.selenium.SeleniumConfig;
-import com.cognizantsoftvision.maqs.selenium.SeleniumTestObject;
-import com.cognizantsoftvision.maqs.selenium.UIWait;
+import com.cognizantsoftvision.maqs.selenium.*;
```

Comment:
Why are we moving to the wild card pattern? - Not sure this is a good idea

Reply:
brought back all imports